From 12ed4b9ab36a84a5082e13004d799c0d70200382 Mon Sep 17 00:00:00 2001
From: Frank Li <Frank.Li@nxp.com>
Date: Wed, 8 Jul 2026 16:08:33 -0400
Subject: [PATCH 06022/15333] ARM: dts: imx6ull-engicam-microgea: remove
 nand-ecc-strength and nand-ecc-step-size

The nand-ecc-strength and nand-ecc-step-size is bigger than 0. It doesn't
make sense to set to 0.

Remove it to fix below DTB_CHECKS warnings:
  arch/arm/boot/dts/nxp/imx/imx6ull-engicam-microgea-bmm.dtb: nand-controller@1806000 (fsl,imx6q-gpmi-nand): nand-ecc-strength: 0 is less than the minimum of 1

Signed-off-by: Frank Li <Frank.Li@nxp.com>
---
 arch/arm/boot/dts/nxp/imx/imx6ull-engicam-microgea.dtsi | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm/boot/dts/nxp/imx/imx6ull-engicam-microgea.dtsi b/arch/arm/boot/dts/nxp/imx/imx6ull-engicam-microgea.dtsi
index 43518bf07602..2c36c248f6bd 100644
--- a/arch/arm/boot/dts/nxp/imx/imx6ull-engicam-microgea.dtsi
+++ b/arch/arm/boot/dts/nxp/imx/imx6ull-engicam-microgea.dtsi
@@ -44,8 +44,6 @@ &gpmi {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_gpmi_nand>;
 	nand-ecc-mode = "hw";
-	nand-ecc-strength = <0>;
-	nand-ecc-step-size = <0>;
 	nand-on-flash-bbt;
 	status = "okay";
 };
-- 
2.55.0

